lib/litle/api.rb

```diff
@@ -4,9 +4,10 @@ class PaymentPlugin < ::Killbill::Plugin::ActiveMerchant::PaymentPlugin
 
       def initialize
         gateway_builder = Proc.new do |config|
-          ::ActiveMerchant::Billing::LitleGateway.new :login => config[:login],
+          ::ActiveMerchant::Billing::LitleGateway.new :login => config[:username],
                                                       :password => config[:password],
-                                                      :merchant_id => config[:merchant_id]
+                                                      :merchant_id => config[:merchant_id],
+                                                      :secure_page_url => config[:secure_page_url]
```

Comment:
Vantiv/Litle have a sandbox and prelive environments, so I needed to make the url configurable.